routes/index.js

```diff
@@ -127,6 +126,66 @@ module.exports = function (passport) {
     });
   });
 
+  router.post('/userManagement/deleteUser/:badge', jsonParser, function (req, res) {
```

Comment:
You can also remove the jsonParser from these routes, I do not think it is necessary here since you are not parsing a form, etc...